src/core/indexer/mod.rs

```diff
@@ -782,7 +782,7 @@ mod test {
     }
 
     #[rstest]
-    fn exfil_infil_bexfil_binfil(
+    fn exfil_infil_bexfil_binfil_size(
```

Comment:
This is an artifact from another time, I think you probably mean to revert this

Reply:
Oh yeah. Sure. Thank you for catch! Just fixed.